digiroad2-oracle/src/main/scala/fi/liikennevirasto/digiroad2/client/viite/ViiteAuthPropertyReader.scala

```diff
@@ -13,6 +13,10 @@ class ViiteAuthPropertyReader {
     Digiroad2Properties.viitePassword
   }
 
+  def getViiteApiKey: String = {
+    Digiroad2Properties.viiteApiKey
+  }
+
   def getAuthInBase64: String = {
```

Comment:
miksi säilyttää näitä?, olisi hyvä poistaa tää osa tai merkata että tämä muut tälläiset on deprecated

Reply:
sä oot nyt korvannut ton vanhan uudella jolloin nää vanhat basic koodit on nyt turha.